### DISCUSSION TITLE
Side won't load after running production script

### DISCUSSION THREAD

**Discussion by Hjortur17** [statamic/cms#7391] [Troubleshooting]
Hi, after I run the production script in the terminal and then deploy the site I get an error in the console. It something related to the site.js and the error code I get is SyntaxError: 61 (I will attach a photo of error). Any idea what this can be?

<img width="1440" alt="Screenshot 2023-01-21 at 17 10 49" src="https://user-images.githubusercontent.com/19752446/213878558-63137d6a-b10d-402d-8a09-95ce8bee50e0.png">

**Comment by edalzell** [comment]
Do you get the error when you build `dev`?
